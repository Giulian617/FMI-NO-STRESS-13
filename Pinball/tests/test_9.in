100 1
26 78 10
65 26 2
18 65 2
62 65 6
23 26 7
38 23 8
24 18 3
48 65 5
36 62 4
77 36 3
30 38 5
16 36 8
14 65 2
3 16 1
6 77 1
85 3 9
55 16 4
80 16 8
35 62 6
94 77 7
40 62 4
11 48 3
46 23 9
89 94 10
74 18 8
84 40 2
54 77 8
86 94 1
59 26 6
5 26 1
67 80 1
53 16 1
70 11 6
31 23 3
60 78 7
98 24 6
63 94 5
34 5 8
100 78 8
25 63 8
82 85 8
49 38 9
90 82 1
21 54 2
57 65 1
2 98 4
19 31 8
39 6 5
73 100 3
37 26 7
41 59 4
15 23 4
45 77 8
56 35 7
51 36 1
64 65 8
79 63 9
42 36 10
43 2 10
58 53 8
88 53 10
76 78 1
92 34 3
17 46 1
20 14 10
9 2 7
44 74 9
69 16 8
10 21 1
93 49 10
81 65 8
13 57 1
47 16 7
83 24 7
7 26 7
22 2 3
96 48 1
1 57 1
68 38 3
4 62 5
61 9 4
50 20 9
91 80 4
29 63 10
52 1 8
95 80 1
32 10 5
66 57 6
99 79 10
28 69 4
75 95 4
97 74 3
27 63 1
71 63 6
12 81 5
72 51 2
8 64 6
87 100 1
33 87 7
