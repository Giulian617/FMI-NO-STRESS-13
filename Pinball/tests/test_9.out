951754395
517793921
660374617
660374617
310112673
732098658
847353202
88590926
11749538
78262539
747188845
198010028
244731736
622313866
65625730
805872533
517793921
597093578
325755815
328955422
619978257
257618926
65625730
62523801
297293053
325737673
907867414
170257286
31151719
85285491
183448149
454426028
691584319
641311046
239448443
608511394
987855314
812261797
691584319
54505705
20219383
855534045
774539460
987586315
425411854
849138782
151547578
725618639
909802480
942702833
842117978
15698405
